libs/interpret-vision/src/lib/VisionExplanationDashboard/Controls/ImageList.tsx

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
+  Text,
+  FocusZone,
+  List,
+  Image,
+  ImageFit,
+  IRectangle
+} from "@fluentui/react";
+import React from "react";
+
+import { imageListStyles } from "./ImageList.styles";
+
+export interface IImageListProps {
+  imageDim: number;
+  openPanel(): void | undefined;
+}
+
+export interface IImageListState {
+  columnCount: number;
+}
+
+export interface IListItem {
+  title: string;
+  image: string;
+}
+
+const RowsPerPage = 3;
+
+export class ImageList extends React.Component<
+  IImageListProps,
+  IImageListState
+> {
+  public constructor(props: IImageListProps) {
+    super(props);
+
+    this.state = {
+      columnCount: 1
+    };
+  }
+
+  public render(): React.ReactNode {
+    const classNames = imageListStyles();
+
+    const items: IListItem[] = [];
+
+    for (let i = 0; i < 20; i++) {
```

Comment:
I assume this will go away eventually, this is just dummy data for now?

Reply:
yup, will go away once i finish the backend implementation